package.json

```diff
@@ -9,7 +9,10 @@
     "bootstrap": "^5.3.2",
     "date-fns": "^2.30.0",
     "formik": "^2.4.5",
+    "moment": "^2.29.4",
     "react": "^18.2.0",
+    "react-big-calendar": "^1.8.4",
+    "react-datepicker": "^4.21.0",
```

Comment:
remove the packages which are not being used

Reply:
removed